db/scripts/index.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Simple SQL running utilities
+ */
+const fs = require('fs');
+const path = require('path');
+const { Client } = require('pg');
```

Comment:
Out of curiosity it there an advantage of using pg.Client here rather than the pg.Pool we have setup? I thought it's preferred to use [pool](https://node-postgres.com/features/pooling#single-query).

Reply:
Ignore 😊